price/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think this file is here by mistake. Same for `package-lock.json`.

Reply:
sorry I don't know what is a content of this file because  I don't write them😅.
when I want to execute  my code on node side . give some error about can't know what is sync and await 
and I search and find should enter this command  "npm i -D @types/node@16" 
when I enter this code some file added to my project .

Reply:
I don't know the reason behind the error, but if you need to add a package, you should run the command in the repo root directory, not inside price directory.